app/views/dashboards/show.html.erb

```diff
@@ -97,6 +97,11 @@
     <div class="column small-12 medium-6">
       <p class="text">Scaling up this priority package would cost less than scaling up a package of all interventions needed reach the WHA targets. However, the targets would not be achieved under this low-cost scale-up scenario.</p>
     </div>
+    <div class="columns small-12">
+      <button class="text -button js-btn-action" data-action="Info:toggle">
```

Comment:
Here, like I mentioned above, if we remove the click action helper, we'll avoid to have global events names in the HTML. So adding a global class like js-modal-action or whatever and then binding that in the helper would work too. What  do you think?